### PR TITLE
Try making build a little bit more resilient

### PR DESCRIPTION
**Pull request by joschi**
* Retry failed downloads (connection refused) and get rid of intermediate files
* Add missing CMAKE_LINKER in Windows CMake toolchain

### CODE REVIEW

File: Toolchain-x86_64-w64-mingw32.cmake [hunk 1]
```diff
@@ -8,6 +8,7 @@ SET(CMAKE_C_COMPILER x86_64-w64-mingw32-gcc)
 SET(CMAKE_CXX_COMPILER x86_64-w64-mingw32-g++)
 SET(CMAKE_RC_COMPILER x86_64-w64-mingw32-windres)
 SET(CMAKE_AR x86_64-w64-mingw32-ar)
+SET(CMAKE_LINKER x86_64-w64-mingw32-ld)
```
Comment (dbouron) [inline Toolchain-x86_64-w64-mingw32.cmake]: Good catch!

File: lib/build.sh [hunk 2]
```diff
@@ -135,29 +129,27 @@ make -j ${JOBS}
...
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    -DBUILD_tools=OFF \
+    -DBUILD_shared=ON \
+    -DBUILD_examples=OFF \
+    -DBUILD_tests=OFF \
+    -DBUILD_doc=OFF \
+    $CHECKOUT/expat-${EXPAT_VERSION}
+make -j ${JOBS}
+make -j ${JOBS} install
+popd
```
Comment (dbouron) [inline lib/build.sh]: Can you please remove this part, in fact it fails when we try to cross compile to Windows 64 bit target

Comment (joschi) [reply lib/build.sh]: @dbouron I've reverted that part.

Comment (dbouron) [reply lib/build.sh]: Thank you

**Pull request merged**